components/team_general_tab/team_general_tab.jsx

```diff
@@ -99,6 +103,22 @@ export default class GeneralTab extends React.Component {
         this.setState({allow_open_invite: openInvite});
     }
 
+    handleAllowedDomainsSubmit = async () => {
+        var state = {serverError: '', clientError: ''};
+
+        var data = {...this.props.team};
+        data.allowed_domains = this.state.allowed_domains;
```

Comment:
Should we do anything to make sure that the allowed_domains are the right format, or is that handled by the server? I'm wondering about things like extra whitespace or invalid domains

Reply:
Now that I've said that, the server should have handling for this since you could otherwise bypass any clientside validation by using the API directly, and it definitely [does have handling even if it won't error out when you go to save the setting](https://github.com/mattermost/mattermost-server/pull/9142/files#diff-a8af9726c30cd52585af3ebda21ce068R66)